neuraxle/steps/data.py

```diff
@@ -0,0 +1,140 @@
+"""
+Data Steps
+====================================
+You can find here steps that take action on data.
+
+..
+    Copyright 2019, Neuraxio Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+..
+    Thanks to Umaneo Technologies Inc. for their contributions to this Machine Learning
+    project, visit https://www.umaneo.com/ for more information on Umaneo Technologies Inc.
+
+"""
+import random
+from typing import Iterable
+
+from neuraxle.base import BaseStep, MetaStepMixin, NonFittableMixin
+from neuraxle.steps.output_handlers import InputAndOutputTransformerMixin
+
+
+class DataShuffler(NonFittableMixin, InputAndOutputTransformerMixin, BaseStep):
+    """
+    Data Shuffling step that shuffles data inputs, and expected_outputs at the same time.
+
+    .. code-block:: python
+
+        p = DataShuffler(
+            EpochRepeater(
+                ForecastingPipeline(),
```

Comment:
Is this still up to date? Are those objects still MetaStepsMixins or are they pipeline steps before the forecast, but at the same level?

Reply:
no datashuffler is only a base step fixed